src/atoms/Img/__tests__/img.spec.js

```diff
@@ -5,6 +5,16 @@ import renderer from 'react-test-renderer';
 import Img from 'atoms/Img';
 
 describe('<Img />', () => {
+  const warn = console.warn;
+
+  beforeAll(() => {
+    console.warn = jest.fn();
+  });
```

Comment:
What's the intention of this?

Reply:
This was warning you that the lazyload module wasn't available -- which it wouldn't be in the tests because we load it at page load.